app/Models/BookingRequest.php

```diff
@@ -97,6 +97,6 @@ public function reviewers()
     }
 
     public function scopePending(Builder $query) {
-        $query->where('status', 'review');
+        $query->where('status', 'review')->orWhere('status', 'pending');
```

Comment:
maybe this should be split into 2 different scopes?

Reply:
i disagree since a booking that is in review is still a pending review

Reply:
sounds fair